### PR TITLE
fix: database

### PR DESCRIPTION
**Pull request by MichaelLELU**

### CODE REVIEW

**Review by camille-haxa — APPROVED** [review]

File: server/database/schema.sql [hunk 1]
```diff
@@ -1,64 +1,53 @@
...
-  is_premium BOOLEAN NOT NULL DEFAULT FALSE,
-  is_free BOOLEAN NOT NULL DEFAULT FALSE,
-  requires_account BOOLEAN NOT NULL DEFAULT FALSE
-);
-
 -- Catégorie
 CREATE TABLE category (
-  id INT UNSIGNED PRIMARY KEY AUTO_INCREMENT NOT NULL,
-  name VARCHAR(80) NOT NULL
+    id INT UNSIGNED PRIMARY KEY AUTO_INCREMENT NOT NULL,
```
Comment (aurelienLavanchy) [inline server/database/schema.sql:3]: Remove unnecessary spaces at the beginning of the line

File: server/database/schema.sql [hunk 2]
```diff
@@ -1,64 +1,53 @@
...
+    name VARCHAR(80) NOT NULL,
+    url VARCHAR(255) NOT NULL,
+    image VARCHAR(255) NOT NULL,
+    description TEXT,
+    date DATE NOT NULL,
+    is_premium BOOLEAN NOT NULL DEFAULT FALSE,
+    is_free BOOLEAN NOT NULL DEFAULT FALSE,
+    requires_account BOOLEAN NOT NULL DEFAULT FALSE,
+    category_id INT UNSIGNED,
+    FOREIGN KEY (category_id) REFERENCES category (id)
```
Comment (aurelienLavanchy) [inline server/database/schema.sql:19]: Unsure if SQL likes the space between `category` and `(id)`
Ditto for the following foreign keys

File: server/database/schema.sql [hunk 3]
```diff
@@ -1,64 +1,53 @@
...
 );
 
 -- Utilisateur
 CREATE TABLE role (
-  id INT UNSIGNED PRIMARY KEY AUTO_INCREMENT NOT NULL,
-  name VARCHAR(80) NOT NULL,
-  access VARCHAR(40) NOT NULL
+    id INT UNSIGNED PRIMARY KEY AUTO_INCREMENT NOT NULL,
```
Comment (aurelienLavanchy) [inline server/database/schema.sql:24]: Unnecessary spaces at the beginning of the line

File: server/database/schema.sql [hunk 4]
```diff
@@ -1,64 +1,53 @@
...
+    FOREIGN KEY (role_id) REFERENCES role (id)
 );
 
 CREATE TABLE payment (
-  id INT UNSIGNED PRIMARY KEY AUTO_INCREMENT NOT NULL,
-  date DATE NOT NULL,
-  type VARCHAR(40) NOT NULL,
-  user_id INT UNSIGNED,
-  FOREIGN KEY(user_id) REFERENCES user(id)
+    id INT UNSIGNED PRIMARY KEY AUTO_INCREMENT NOT NULL,
```
Comment (aurelienLavanchy) [inline server/database/schema.sql:41]: Remove unnecessary spaces at the beginning

File: server/database/schema.sql [hunk 5]
```diff
@@ -1,64 +1,53 @@
...
+    user_id INT UNSIGNED,
+    FOREIGN KEY (user_id) REFERENCES user (id)
 );
 
 CREATE TABLE privilege (
-  video_id INT UNSIGNED,
-  user_id INT UNSIGNED,
-  FOREIGN KEY(video_id) REFERENCES video(id),
-  FOREIGN KEY(user_id) REFERENCES user(id)
+    video_id INT UNSIGNED,
```
Comment (aurelienLavanchy) [inline server/database/schema.sql:49]: Remove unnecessary spaces at the beginning (yet again)

**Review by aurelienLavanchy — CHANGES_REQUESTED** [review]

**Review by aurelienLavanchy — APPROVED** [review]

**Review by ioayoub — APPROVED** [review]

**Pull request merged**